src/box/memtx_space.c

```diff
@@ -1037,6 +1096,29 @@ memtx_build_on_replace(struct trigger *trigger, void *event)
 		if (stmt->old_tuple != NULL)
 			tuple_unref(stmt->old_tuple);
 	}
+	/*
+	 * Set on_rollback trigger on stmt to avoid
+	 * problem when rollbacked changes appears in
+	 * built-in-background index.
+	 */
+	struct on_rollback_trigger_with_data *on_rollback_associates =
+		region_aligned_alloc(&in_txn()->region,
+			sizeof(struct on_rollback_trigger_with_data),
+			alignof(struct on_rollback_trigger_with_data));
+	if (on_rollback_associates == NULL) {
+		diag_set(OutOfMemory,
+			 sizeof(struct on_rollback_trigger_with_data),
+			 "region_aligned_alloc",
+			 "memtx_build_on_replace");
```

Comment:
Nit: the last arg for OOM error is the name of object we are failed to allocate memory. That is the correct form would be "struct on_rollback_trigger_with_data"